mailscanner/sf_version.php

```diff
@@ -93,7 +93,8 @@
     if (get_conf_var('MTA', true) === 'postfix') {
         echo '<br>' . "\n";
         echo 'Postfix ' . __('version11') . ' ';
-        passthru("/usr/sbin/postconf -d | grep 'mail_version =' | cut -d' ' -f3");
+        exec("which postconf", $postconf);
```

Comment:
Hello @asuweb!

You can may be mod the others tests in sf_version.php with "which" command?

Reply:
Yes, that's not a bad idea as it would prevent any other path issues that have yet to be discovered